From patches Mon Sep 17 00:00:00 2001
From: Michael Tokarev <mjt@tls.msk.ru>
To: Gerd Hoffmann <kraxel@redhat.com>, Michael Tokarev <mjt@tls.msk.ru>
Cc: qemu-trivial@nongnu.org, Peter Maydell <peter.maydell@linaro.org>,
 Gerd Hoffmann <kraxel@redhat.com>, qemu-devel@nongnu.org,
 Michael Tokarev <mjt@tls.msk.ru>
Date: Thu, 18 Apr 2013 15:17:33 +0400
Message-id: 1366283853-31785-1-git-send-email-mjt@msgid.tls.msk.ru
Subject: [PATCH 1/1] allow usage of older libusb which didn't define
 LIBUSB_LOG_LEVEL_WARNING

Older libusb does not have any LIBUSB_LOG_LEVEL_* definitions.
Since we expose these values as integers anyway, just use the
right define if the constant isn't defined.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
---
 hw/usb/host-libusb.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/hw/usb/host-libusb.c b/hw/usb/host-libusb.c
index 29f35b3..78c315f 100644
--- a/hw/usb/host-libusb.c
+++ b/hw/usb/host-libusb.c
@@ -1251,6 +1251,9 @@ static Property usb_host_dev_properties[] = {
     DEFINE_PROP_UINT32("isobufs",  USBHostDevice, iso_urb_count,    4),
     DEFINE_PROP_UINT32("isobsize", USBHostDevice, iso_urb_frames,   32),
     DEFINE_PROP_INT32("bootindex", USBHostDevice, bootindex,        -1),
+#ifndef LIBUSB_LOG_LEVEL_WARNING /* older libusb didn't define these */
+#define LIBUSB_LOG_LEVEL_WARNING 2
+#endif
     DEFINE_PROP_UINT32("loglevel",  USBHostDevice, loglevel,
                        LIBUSB_LOG_LEVEL_WARNING),
     DEFINE_PROP_BIT("pipeline",    USBHostDevice, options,
-- 
1.7.10.4


